_typos.toml

```diff
@@ -0,0 +1,14 @@
+[default]
+extend-ignore-identifiers-re = [
+    # Related to serialized object
+    "ser.*",
+    # Used when dumping tfhe-rs parameters set into Sage format
+    "ND.*",
+    # Related to FHE strings example handling "banana"
+    "ba",
+    ".*_ba",
```

Comment:
what is this one for ?

Reply:
there are variables `enc_ba` for encrypted `banana`.

Reply:
could we hardcode the enc_ba then ?

I prefer avoiding wildcards if we can to avoid false negatives